### PR TITLE
feat(instance): add ip testing

### PR DESCRIPTION
**Pull request by remyleone**

### CODE REVIEW

File: internal/namespaces/instance/v1/ip_test.go [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+	t.Run("Simple", core.Test(&core.TestConfig{
+		Commands: GetCommands(),
+		Cmd:      "scw instance ip get",
+		Check:    core.TestCheckGolden(),
+	}))
+}
+
+func Test_IpList(t *testing.T) {
+	t.Run("Usage", core.Test(&core.TestConfig{
```
Comment (loicbourgois) [inline internal/namespaces/instance/v1/ip_test.go]: we should remove this test

all usage are tested in main

see https://github.com/scaleway/scaleway-cli/blob/v2/cmd/scw/main_test.go

File: internal/namespaces/instance/v1/testdata/test-ip-create-usage.stderr.golden [hunk 2]
```diff
@@ -0,0 +1,28 @@
+Reserve an IP.
```
Comment (loicbourgois) [inline internal/namespaces/instance/v1/testdata/test-ip-create-usage.stderr.golden]: remove this file as usage are all tested in main

File: internal/namespaces/instance/v1/testdata/test-ip-delete-usage.stderr.golden [hunk 3]
```diff
@@ -0,0 +1,26 @@
+Delete the IP with the given ID.
```
Comment (loicbourgois) [inline internal/namespaces/instance/v1/testdata/test-ip-delete-usage.stderr.golden]: remove this file as usage are all tested in main

File: internal/namespaces/instance/v1/testdata/test-ip-get-usage.stderr.golden [hunk 4]
```diff
@@ -0,0 +1,26 @@
+Get details of an IP with the given ID or address.
```
Comment (loicbourgois) [inline internal/namespaces/instance/v1/testdata/test-ip-get-usage.stderr.golden]: remove this file as usage are all tested in main

File: internal/namespaces/instance/v1/testdata/test-ip-list-usage.stderr.golden [hunk 5]
```diff
@@ -0,0 +1,24 @@
+List IPs.
```
Comment (loicbourgois) [inline internal/namespaces/instance/v1/testdata/test-ip-list-usage.stderr.golden]: remove this file as usage are all tested in main

File: internal/namespaces/instance/v1/testdata/test-ip-update-usage.stderr.golden [hunk 6]
```diff
@@ -0,0 +1,28 @@
+Update IP.
```
Comment (loicbourgois) [inline internal/namespaces/instance/v1/testdata/test-ip-update-usage.stderr.golden]: remove this file as usage are all tested in main

File: internal/namespaces/instance/v1/ip_test.go [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+		Commands: GetCommands(),
+		Cmd:      "scw instance ip create",
+		Check:    core.TestCheckGolden(),
+	}))
+}
+
+func Test_IpDelete(t *testing.T) {
+	t.Run("Simple", core.Test(&core.TestConfig{
+		Commands: GetCommands(),
+		Cmd:      "scw instance ip delete",
```
Comment (QuentinBrosse) [inline internal/namespaces/instance/v1/ip_test.go]: Invalid command (check the golden)

File: internal/namespaces/instance/v1/ip_test.go [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+		Check:    core.TestCheckGolden(),
+	}))
+}
+
+func Test_IpGet(t *testing.T) {
+	t.Run("Simple", core.Test(&core.TestConfig{
+		Commands: GetCommands(),
+		Cmd:      "scw instance ip get",
```
Comment (QuentinBrosse) [inline internal/namespaces/instance/v1/ip_test.go]: Invalid command (check the golden)

File: internal/namespaces/instance/v1/ip_test.go [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+		Commands: GetCommands(),
+		Cmd:      "scw instance ip list",
+		Check:    core.TestCheckGolden(),
+	}))
+}
+
+func Test_IpUpdate(t *testing.T) {
+	t.Run("Simple", core.Test(&core.TestConfig{
+		Commands: GetCommands(),
+		Cmd:      "scw instance ip update",
```
Comment (QuentinBrosse) [inline internal/namespaces/instance/v1/ip_test.go]: Invalid command (check the golden)

File: internal/namespaces/instance/v1/ip_test.go [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+
+	"github.com/scaleway/scaleway-cli/internal/core"
+	"github.com/scaleway/scaleway-sdk-go/api/instance/v1"
+)
+
+func Test_IpCreate(t *testing.T) {
+	t.Run("Simple", core.Test(&core.TestConfig{
+		Commands: GetCommands(),
+		Cmd:      "scw instance ip create",
+		Check:    core.TestCheckGolden(),
```
Comment (loicbourgois) [inline internal/namespaces/instance/v1/ip_test.go]: most `Check` should have a `TestCheckExitCode`

we can use `TestCheckCombine` to have multiple checks

**Pull request merged**